views/partials/header.ejs

```diff
@@ -29,12 +40,33 @@
                         <a class="nav-link btn btn-sm btn-danger mx-2" href="/register">Register</a>
                     </li>
                     <% } else { %>
+
+
+                        <li class="dropdown">
+                          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                            <i class="fa fa-user-circle-o" style="color:red"></i> &nbsp; <%=currentUser.username %> &nbsp;<span class="caret"></span></a>
+                            <!-- added dropdownmenu-->
+                          <ul class="dropdown-menu">
+
+                            <li><a href="#">Post &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="badge">1</span></a></li>
+                            <li role="separator" class="divider"></li>
+                            <li><a href="#">Setting</a></li>
+                            <li role="separator" class="divider"></li>
+                            <li><a href="#">Last Post &nbsp;&nbsp;<span class="badge">2 hour before</span></a></li>
+
```

Comment:
This is supposed to be dynamic and rendered from server side and thus I gave `intermediate` label. Not a static one.

I'll change the issue to `diificult` label.